csharp/src/Ice/IObjectPrx.cs

```diff
@@ -503,6 +513,16 @@ public void IceUpdateRequestHandler(IRequestHandler? previous, IRequestHandler?
             }
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected static async Task IceAwaitResponse(Task<IncomingResponseFrame> task,
```

Comment:
async methods and more generally any method that returns a Task or ValueTask needs an Async suffix. So it should be `IceAwaitResponseAsync` or probably `IceReadResponseAsync`: the direct caller won't wait for anything so let's not emphasize (a)wait.